.github/workflows/publish-image.yml

```diff
@@ -0,0 +1,49 @@
+name: Docker Image Build and Publish
+
+on:
+  push:
+    branches: [ "main" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
The image URL should resolve with the following URI: `quay.io/argoprojlabs/argocd-extension-installer:<tag>`
We need to add the prefix `argoprojlabs/` in the image name.